<!DOCTYPE HTML>
<html>
<head>
    <title>Michelle Lynn Gill</title>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link rel="stylesheet" href="assets/css/main.css" />
    <link rel="stylesheet" href="assets/css/kube.min.css" />
    <link rel="icon" href="images/profile_bw.jpg">
</head>

<body>

<!-- Nav -->
<nav id="nav" class="no-print">
<ul class="container">
    <li><a href="#bio">Bio</a></li>

    <li><a href="#education">Education</a></li>
    <li><a href="#experience">Experience</a></li>

    <li><a href="#presentations">Presentations</a></li>
    <li><a href="#projects">Projects</a></li>

    <li><a href="#news">Honors</a></li>
    <li><a href="#publications">Publications</a></li>
    <li><a href="#contact">Contact</a></li>

    <li><a href="https://resume.michellelynngill.com">Resume</a></li>
</ul>
</nav>

<!-- Intro -->
<div class="wrapper style1 first">
<article class="container" id="bio">

<div class="row">

<div class="4u 12u(mobile) no-print">
    <span class="image fit"><img src="images/profile_bw.jpg" alt="" style="border-radius: 50%"/></span>
</div>

<div class="8u 12u(mobile)">
<header>
    <h1 class="no-print"><strong>Michelle Lynn Gill</strong></h1>
    <h1 class="print-only"><strong>Michelle Lynn Gill, Ph.D.</strong></h1>
</header>

<p class="no-print">
I am currently a Senior AI and Deep Learning Scientist in Life Sciences and the Tech Lead for <a href="https://www.nvidia.com/en-us/clara/drug-discovery/" target="_blank">Clara Discovery</a> at NVIDIA. 
I lead a growing team of researchers and engineers who use use data, deep learning, and HPC to develop tools that accelerate drug discovery.<br/><br/>
Previously I was a Senior Machine Learning Engineer and Data Scientist at BenevolentAI 
and a member of <a href="https://www.biochem.cumc.columbia.edu/research-labs/palmer-lab" target="_blank">Arthur Palmer's</a> 
research group where I developed and applied nuclear magnetic resonance (NMR) spin relaxation experiments to 
understand how enzyme dynamics are critical to biological function.<br/><br/>
This website serves as my professional CV and has been formatted to export as a PDF. My <a href="https://resume.michellelynngill.com/" target="_blank">resume</a> 
and <a href="#contact">contact</a> information are also available.<br/>
</p>

</div>
</div>

</article>
</div>

<!-- Education -->
<div class="wrapper style2">
<article id="education" class="container 75%">
<header>
    <h2>Education</h2>
</header>

<p>
<b>Ph.D., Molecular Biophysics & Biochemistry</b>, 2003–2006<br/>
Yale University, New Haven, CT<br/>
Thesis: <b>Development of <sup>205</sup>Tl NMR methods for the direct study of monovalent metal ions and ligands in nucleic acids</b>
<a href="publications/GillMichelle_Thesis.pdf" target="_blank" style="text-decoration:none">
    <span class="label label-outline label-blue">THESIS</span></a>
<a href="presentations/2006_GillMichelle_ThesisDefense.pdf" target="_blank" style="text-decoration:none;">
    <span class="label label-outline label-yellow">DEFENSE</span></a>
</br>
Advisors: <i>J. Patrick Loria and Scott Strobel</i><br/><br/>
</p>

<p>
<b>M.Phil., Molecular Biophysics & Biochemistry</b>, 2001–2003<br/>
Yale University, New Haven, CT<br/><br/>
</p>

<p>
<b>B.S., Biochemistry</b>, 1997-2001</br>
Highest Distinction and Honors (<i>Summa Cum Laude</i>)<br/>
University of Kansas, Lawrence, KS<br/>
</p>

</article>
</div>

<!-- Experience -->
<div class="wrapper style3">
<article id="experience" class="container 75%">
<header>
    <h2>Experience</h2>
</header>

<p>
<b>Tech Lead, Clara Discovery</b>, 2019-Present<br/>
<b>Senior AI and Deep Learning Scientist, Cheminformatics and Proteomics</b>, 2019-Present<br/>
<b>Senior Deep Learning Consultant</b>, 2017-2018<br/>
<b>NVIDIA</b><br/>
I serve as the Tech Lead for Clara Discovery, NVIDIA's platform for accelerating the drug discovery process through molecular dynamics, HPC, and deep learning. 
My responsibilities include managing both internal research and product cycle planning, and external collaborations with researchers in pharma, biotech, and academia. I am also the lead developer of MegaMolBART, a transformer-based seq2seq cheminformatics model trained at scale for drug discovery tasks.
<br/><br/>
Earlier work focused on proteomics, including development of deep learning models to predict peptide spectral matches (PSMs) in proteomics sequencing with >95% F1. 
I also led a team that used GCNNs to predict molecular properties and finished <a href="https://www.kaggle.com/c/champs-scalar-coupling/leaderboard">33rd in a Kaggle competition</a>.
<br/><br/>
As a deep learning consultant, I assisted clients in the pharmaceutical and materials science space in utilizing deep learning for strategic advantage. I helped develop proof of concept deep learning experiments and pipelines to validate approach and to identify technology stack and engineering architecture for solution deployment.<br/><br/>
</p>

<p>
<b>Senior Machine Learning Engineer</b>, 2019<br/>
<b>Senior Data Scientist</b>, 2018-2019<br/>
<b>BenevolentAI</b><br/>
Utilized scientific and machine learning methods to assist with all stages of the drug discovery process, from target identification to chemistry validation. Select focuses of my work have included using matrix factorization and graph convolutional neural networks (GCNNs) to determine the importance of drug mechanisms in knowledge graphs, and deep learning (3D CNNs) and cheminformatics methods to predict ligand pose and affinity within a target.<br/><br/>
</p>

<p>
<b>Senior Data Scientist</b>, 2016-2017<br/>
<b>Metis</b><br/>
Designed and created Spark machine learning and NLP curriculum using self-made Docker containers. Conducted corporate trainings focused on Python and Spark. Developed 12-week machine learning course for F100 company. Co-instructed 12-week data science bootcamps. Developed and conducted take home coding exercise to assist with interview preparation.<br/><br/>
</p>

<p>
<b>Scientist</b>, 2014-2016<br/>
<b>National Cancer Institute, National Institutes of Health</b><br/>
Developed parallelized, compressed sensing methods for reconstruction of non-uniformly sampled NMR data.<br/><br/>
</p>

<p>
<b>Postdoctoral Research Fellow</b>, 2008-2014<br/>
<b>Columbia University, Department of Biochemistry and Molecular Biophysics</b><br/>
Part of a collaboration that demonstrated conformational selection is critical in the highly concerted mechanism of the DNA methyltransferase, AlkB. Developed multiple quantum NMR spin relaxation experiments for quantifying the slow timescale (microsecond – millisecond) motions of methyl sidechains.<br/>
Advisor: <i>Professor Arthur G. Palmer, III</i><br/><br/>
</p>

<p>
<b>Consultant</b>, 2006–2007<br/>
<b>The Boston Consulting Group</b><br/>
Worked with clients in the finance and pharmaceutical sectors to streamline organizational structure and identify novel investment opportunities. I was part of the case team that won the 2007 Global BCG Strategy Olympics for our work with a pharmaceutical client.<br/>
</p>

</article>
</div>

<!-- Presentations -->
<div class="wrapper style2">
<article id="presentations" class="container 75%">
<header>
    <h2>Presentations</h2>
</header>

<!-- <p>
    <b>XXX</b>
    <a href="presentations/XXX.pdf" target="_blank" style="text-decoration:none;">
    	<span class="label label-outline label-yellow">SLIDES</span></a>
    <br/>
    <b><u>Michelle Gill</u></b><br/>
    <i>IBM Research</i>, Invited Talk, January XX, 2022, Virtual<br/><br/>
</p> -->
	
<p>
    <b>Exploring Molecular Space and Accelerating Drug Discovery on the GPU with Clara Discovery</b>
    <a href="presentations/2021_GillMichelle_GatesFoundationGrandChallenges.pdf" target="_blank" style="text-decoration:none;">
    	<span class="label label-outline label-yellow">SLIDES</span></a>
    <br/>
    <b><u>Michelle Gill</u></b><br/>
    <i>Gates Foundation Grand Challenges: Applications of Artifical Intelligence in Machine Learning</i>, Invited Talk, November 10, 2021, Virtual<br/><br/>
</p>
	
<p>
    <b>Accelerating Drug Discovery with Clara Discovery and MegaMolBART</b>
    <a href="presentations/2021_GillMichelle_RSCAIChemistry.pdf" target="_blank" style="text-decoration:none;">
    	<span class="label label-outline label-yellow">SLIDES</span></a>
    <br/>
    <b><u>Michelle Gill</u></b><br/>
    <i>4th RSC-BMCS / RSC-CICAG Artificial Intelligence in Chemistry</i>, Invited Talk, September 27, 2021, Virtual<br/><br/>
</p>
	
<p>
    <b>Accelerating Drug Discovery with Clara Discovery's MegaMolBart</b>
    <a href="https://www.youtube.com/watch?v=sJd670Y2DXI" target="_blank" style="text-decoration:none;">
	    <span class="label label-outline label-black">VIDEO</span></a>
    <br/>
    <b><u>Michelle Gill</u></b><br/>
    <i>Demo</i>, April 12, 2021<br/><br/>
</p>

<p>
    <b>Real Time, GPU-Accelerated Analysis and Visualization in the Life Sciences</b>
    <a href="https://2020ricedsconference.rice.edu/speakers/" target="_blank" style="text-decoration:none;">
        <span class="label label-outline label-black">SPEAKERS</span></a>
    <a href="press/GillMichelle_RiceDataScienceConference_Abstract_2020.pdf" target="_blank" style="text-decoration:none;">
        <span class="label label-outline label-orange">ABSTRACT</span></a>
    <a href="presentations/2020_GillMichelle_RiceDataScienceConference.pdf" target="_blank" style="text-decoration:none;">
    	<span class="label label-outline label-yellow">SLIDES</span></a>
    <br/>
    <b><u>Michelle Gill</u></b> and Avantika Lal<br/>
    <i>Ken Kennedy Institute Data Science Conference</i>, Invited Keynote, October 26-27, 2020, Virtual<br/><br/>
</p>

<p>
<b>Artificial intelligence driven drug discovery</b>
<a href="presentations/2019_GillMichelle_NYC_R_Keynote.pdf" target="_blank" style="text-decoration:none;">
    <span class="label label-outline label-yellow">SLIDES</span></a>
<br/>
<i>NYC R Conference</i>, Invited Presentation, May 10, 2019, New York, NY<br/><br/>
</p>

<p>
<b>Panel: Careers in data science</b>
<a href="https://michellelynngill.com/press/GillMichelle_TriICareerSymposiumFlyer_2019.pdf" target="_blank" style="text-decoration:none;">
    <span class="label label-outline label-orange">PROGRAM</span></a>
<br/>
<i>Tri-Institutional Career Symposium</i>, April 9, 2019, New York, NY<br/>
Memorial Sloan Kettering Cancer Center, The Rockefeller University and Weill Cornell Medicine<br/></br>
</p>

<p>				
<b>Machine learning for target identification and lead optimization in drug discovery</b>
<a href="https://www.meetup.com/New-York-Area-Group-for-Informatics-and-Modeling/events/258069925/" target="_blank" style="text-decoration:none;">
    <span class="label label-outline label-orange">ABSTRACT</span></a>
<br/>
Alix Lacoste and <b><u>Michelle Gill</u></b><br/>
<i>New York Area Group for Informatics and Modeling</i>, Invited Presentation, February 26, 2019, New York, NY<br/><br/>
</p>

<p>
<b>Accelerating the journey from data to medicine</b>
<a href="https://nips.cc/Expo/Conferences/2018/Schedule?talk_id=20" target="_blank" style="text-decoration:none;">
    <span class="label label-outline label-orange">ABSTRACT</span></a>
<br/>
Amir Saffari, Dan Neil, Alix Lacoste, and <b><u>Michelle Gill</u></b><br/>
<i>NeurIPS</i>, Expo Talk, 2018, Montreal, Canada<br/><br/>
</p>

<p>
<b>Artificial intelligence as a catalyst for scientific discovery</b>
<a href="https://conferences.oreilly.com/jupyter/jup-ny/public/schedule/speaker/264851" target="_blank" style="text-decoration:none;">
    <span class="label label-outline label-orange">ABSTRACT</span></a><br/>
<a href="presentations/2018_GillMichelle_JupyterCon_Keynote.pdf" target="_blank" style="text-decoration:none;">
    <span class="label label-outline label-yellow">SLIDES</span></a>
<a href="https://www.oreilly.com/library/view/jupytercon-new-york/9781492025818/video322559.html" target="_blank" style="text-decoration:none;">
    <span class="label label-outline label-black">VIDEO</span></a><br/>
<i>JupyterCon</i>, Invited Keynote, 2018, New York, NY<br/><br/>
</p>

<p>
<b>From structural biology to AI: a holistic approach to studying molecular machines</b>
<a href="presentations/2018_GillMichelle_Brookhaven.pdf" target="_blank" style="text-decoration:none;">
    <span class="label label-outline label-yellow">SLIDES</span></a><br/>
<i>Brookhaven National Laboratory</i>, Invited Presentation, 2018, Upton, NY<br/><br/>
</p>

<p>
<b>Efficient image search and identification: the making of Wine-O.AI</b>
<a href="presentations/2017_GillMichelle_SciPy_WINE-O_AI.pdf" target="_blank" style="text-decoration:none;">
    <span class="label label-outline label-yellow">SLIDES</span></a>
<a href="https://www.youtube.com/watch?v=3CUWMZ1304I" target="_blank" style="text-decoration:none;">
    <span class="label label-outline label-black">VIDEO</span></a>
<a href="https://github.com/mlgill/wine-o.ai" target="_blank" style="text-decoration:none;">
    <span class="label label-outline label-green">CODE</span></a><br/>
<i>SciPy Conference</i>, Selected Presentation, 2017, Austin, TX<br/><br/>
</p>

<p>
<b>Learning from text: natural language processing with python</b>
<a href="presentations/2017_GillMichelle_ODSC_PythonNLP.pdf" target="_blank" style="text-decoration:none;">
    <span class="label label-outline label-yellow">SLIDES</span></a>
<a href="https://github.com/mlgill/ODSC_East_2017_PythonNLP" target="_blank" style="text-decoration:none;">
    <span class="label label-outline label-green">CODE</span></a><br/>
<i>ODSC East</i>, Tutorial, 2017, Boston, MA<br/>
</p>

</article>
</div>


<!-- Projects -->
<div class="wrapper style3 no-print">
<article id="projects" class="container 75%">
<header>
    <h2>Projects</h2>
</header>

<p>
<b>COVID-19 Biohackathon</b>
<a href="https://github.com/virtual-biohackathons/covid-19-bh20/wiki" target="_blank" style="text-decoration:none;">
    <span class="label label-outline label-orange">WEBSITE</span></a>
<a href="https://github.com/SWISS-MODEL/covid-19-Annotations-on-Structures" target="_blank" style="text-decoration:none;">
    <span class="label label-outline label-green">CODE</span></a><br/>
April 5-11, 2020<br/>
A biohackathon focused on developing tools for COVID-19 research and drug discovery. 
I was part of a team that developed annotations for 3D protein structures. 
My contribution was an automated method for calculating and visualizing solvent accessibility on families of protein strcutures.
<br/><br/>
</p>

<p>
<b>themodernscientist</b>
<a href="http://themodernscientist.com" target="_blank" style="text-decoration:none;">
    <span class="label label-outline label-orange">WEBSITE</span></a>
<a href="https://github.com/modernscientist/modernscientist.github.com" target="_blank" style="text-decoration:none;">
    <span class="label label-outline label-green">CODE</span></a><br/>
My personal blog discusses data visualization, optimization, automation, and various other computationally-related interests. 
Posts frequently incorporate Jupyter notebooks and shell scrips. 
The blog itself is created using Pelican, a Python-based static blogging engine, and the website is hosted on GitHub.
<br/><br/>
</p>

<p>
<b>NESTA-NMR</b>
<a href="publications/GillMichelle_JBNMR_2015.pdf" target="_blank" style="text-decoration:none;">
    <span class="label label-outline label-blue">PAPER</span></a>
<a href="http://nestanmr.com/" target="_blank" style="text-decoration:none;">
    <span class="label label-outline label-orange">WEBSITE</span></a>
<a href="http://docs.nestanmr.com/" target="_blank" style="text-decoration:none;">
    <span class="label label-outline label-cyan">DOCS</span></a><br/>
Developed and currently maintain a program, called NESTA-NMR, that enables the acquisition of experimental data to be completed up to 100X faster. 
This project uses a compressed sensing algorithm, called NESTA. The program is written in C, highly parallelized, and best-in-class for speed and optimization accuracy. Associated scientific (GSL and FFTW) and parallelization (pthreads) libraries were used. I also built the website using Flask and created the documentation using Sphinx.<br/><br/>
</p>

<!-- <p>
<b>pdLSR</b>
<a href="https://github.com/mlgill/pdLSR" target="_blank" style="text-decoration:none;"><span class="label label-outline label-green">CODE</span></a>
<a href="http://themodernscientist.com/posts/2016/2016-07-17-pdLSR_pandas_aware_least_squares_regression/" target="_blank" style="text-decoration:none;"><span class="label label-outline label-orange">BLOG</span></a>
<br/>
A library for performing least squares regression. It attempts to seamlessly incorporate this task in a Pandas-focused workflow. Input data are expected in dataframes, and multiple regressions can be performed using functionality similar to Pandas groupby.<br/><br/>
</p> -->

<!-- <p>
<b>MFOutParser</b>
<a href="https://github.com/mlgill/mfoutparser" target="_blank" style="text-decoration:none;"><span class="label label-outline label-green">CODE</span></a><br/>
A Python library developed during my postdoctoral research that parses a particularly challenging text format and performs preliminary analysis on the resulting data using Pandas.<br/>
</p> -->

</article>
</div>


<!-- News and Misc -->
<div class="wrapper style2">
<article id="news" class="container 75%">
<header>
    <h2>Honors & Awards</h2>
</header>

<h3>Service</h3>

<p>
<b>Judge</b></br>
<i>NYC STEM Fair</i>, 2022</br>
Evaluating preliminary and final round submissions in the biochemistry track.</br></br>
</p>

<p>
<b>Program Chair</b></br>
<i>PyData NYC</i>, 2018</br>
Responsible for conference content including: recruitment of proposal reviewers, soliciation and aggregation of proposal feedback, selection of presentations, notification of selected speakers, and scheduling of talks.</br></br>
</p>

<p>
<b>Machine Learning Symposium Co-Chair</b></br>
<i>SciPy Conference</i>, 2018</br>
Co-chair of machine learning / deep learning symposium at SciPy 2018. Responsible for recruiting reviewers, reviewing proposals, selecting talks, and running symposium.</br></br>
</p>

<p>
<b>Proposal Reviewer</b></br>
<i>JupyterCon</i>, 2018</br>
Responsible for reviewing and scoring proposal submissions.</br></br>
</p>

<p>
<b>Reviewer</b></br>
<i>Journal of Open Source Software (JOSS)</i>, 2018–2019</br>
Review submissions to JOSS within my expertise area as needed.</br>
</p>

<h3>Awards</h3>

<p>
NIH Postdoctoral Research Fellowship (<i>F32 GM089047</i>), 2009–2012</br>
Global BCG Strategy Olympics, Winning Team, 2007</br>
NSF Graduate Research Fellowship, 2002–2006</br>
Outstanding Graduate Teaching Assistant, 2003</br>
Barry M. Goldwater Scholar, 2000–2001</br>
Outstanding Undergraduate Honors Research Thesis, 2001</br>
Kansas Board of Regents Full Tuition Merit Scholarship, 1997–2001</br>

<!-- <p class="print-only"></br></br></p> -->
</p>

<h3>Press</h3>

<p>
<b>Deep Learning at NVIDIA (with Michelle Gill)
</b>
<a href="https://www.datacamp.com/community/podcast/deep-learning-nvidia" target="_blank" style="text-decoration:none;"><span class="label label-outline label-black">AUDIO</span></a></br>
<i>DataFramed</i>, DataCamp, Podcast Interview, 2018</br>
Interview about my deep learning work at NVIDIA</br></br>
</p>

<p>
<b>How to Keep Your Job Regardless of AI</b>
<a href="https://www.ibtimes.com/how-keep-your-job-regardless-ai-2624078" target="_blank" style="text-decoration:none;"><span class="label label-outline label-orange">ARTICLE</span></a></br>
<i>International Business Times</i>, 2017</br>
Profile of my transition from biophysicist to data scientist</br>
</p>

</article>
</div>


<!-- Publications -->
<div class="wrapper style3">
<article id="publications" class="container 75%">
<header>
    <h2>Publications</h2>
</header>

<!-- <a href="XX" target="_blank" style="text-decoration:none;"><span class="label label-outline label-purple">DOI</span></a>
<a href="XX" target="_blank" style="text-decoration:none;"><span class="label label-outline label-blue">PDF</span></a> 
<a href="XX" target="_blank" style="text-decoration:none;"><span class="label label-outline label-yellow">SLIDES</span></a>
<a href="XX" target="_blank" style="text-decoration:none;"><span class="label label-outline label-red">POSTER</span></a>
<a href="XX" target="_blank" style="text-decoration:none;"><span class="label label-outline label-black">VIDEO</span></a>
<a href="XX" target="_blank" style="text-decoration:none;"><span class="label label-outline label-green">CODE</span></a> 
<a href="XX" target="_blank" style="text-decoration:none;"><span class="label label-outline label-orange">LINK</span></a> -->

<p>
<u><b>Gill, M.L.</u></b>
<b>The Rise of the Machines in Chemistry</b>, Invited Prospectus<br/>
	<i>Magnetic Resonance in Chemistry</i>, <i>Accepted for forthcoming special issue</i><br/>
	
<p>
<u><b>Gill, M.L.</u></b>, Hsu, A., Palmer, A.G.
<a href="https://doi.org/10.26434/chemrxiv.7762067" target="_blank" style="text-decoration:none;"><span class="label label-outline label-orange">ChemRxiv</span></a>
<a href="https://doi.org/10.1007/s10858-019-00240-w" target="_blank" style="text-decoration:none;"><span class="label label-outline label-purple">DOI</span></a>
<a href="publications/GillMichelle_JBNMR_2019.pdf" target="_blank" style="text-decoration:none;"><span class="label label-outline label-blue">PDF</span></a><br/>
<b>Detection of chemical exchange in methyl groups of macromolecules</b><br/>
	<i>Journal of Biomolecular NMR</i>, 2019, 73, 443-450<br/>
Poster presented at <i>57<sup>th</sup> Experimental Nuclear Magnetic Resonance Conference</i>, 2016, Pittsburgh, PA
<a href="posters/GillMichelle_MethylRex_ENC_2016.pdf" target="_blank" style="text-decoration:none;"><span class="label label-outline label-red">POSTER</span></a><br/><br/>
</p>

<p>
Tong, M., Pelton, J., <u><b>Gill, M.L.</u></b>, Zhang, W., Picart, F., Seeliger, M.
<a href="https://doi.org/10.1038/s41467-017-02240-6" target="_blank" style="text-decoration:none;"><span class="label label-outline label-purple">DOI</span></a>
<a href="publications/GillMichelle_NatureComm_2017.pdf" target="_blank" style="text-decoration:none;"><span class="label label-outline label-blue">PDF</span></a><br/>
<b>Survey of solution dynamics in Src kinase reveals cross talk between the ligand binding and regulatory sites</b><br/>
<i>Nature Communications</i>, 2017, 8, 2160<br/><br/>
</p>

<p>
<u><b>Gill, M.L.</u></b>, Byrd, R.A., Palmer, A.G.
<a href="https://doi.org/10.1039/c5cp06197k" target="_blank" style="text-decoration:none;"><span class="label label-outline label-purple">DOI</span></a>
<a href="publications/GillMichelle_PCCP_2015.pdf" target="_blank" style="text-decoration:none;"><span class="label label-outline label-blue">PDF</span></a><br/>
<b>Dynamics of GCN4 facilitate DNA interaction: a model-free analysis of an intrinsically disordered region</b><br/>
<i>Physical Chemistry and Chemical Physics</i>, 2016, 18, 5839–5849<br/>
Posters presented at <i>International Conference of Magnetic Resonance in Biological Sciences</i>, 2014, Dallas, TX
<a href="posters/GillMichelle_ICMRBS_2014.pdf" target="_blank" style="text-decoration:none;"><span class="label label-outline label-red">POSTER</span></a><br/>
and <i>57<sup>th</sup> Experimental Nuclear Magnetic Resonance Conference</i>, 2016, Pittsburgh, PA
<a href="posters/GillMichelle_GCN4_ENC_2016.pdf" target="_blank" style="text-decoration:none;"><span class="label label-outline label-red">POSTER</span></a><br/><br/>
</p>

<p>
<u><b>Gill, M.L.</u></b>, Sun, S., Li, Y., Byrd, R.A.<br/>
<b>NESTA-NMR: efficient and quantitative processing of multidimensional NUS NMR data</b><br/>
Poster presented at <i>57th Experimental Nuclear Magnetic Resonance Conference</i>, 2016, Pittsburgh, PA
<a href="posters/GillMichelle_NESTA_ENC_2016.pdf" target="_blank" style="text-decoration:none;"><span class="label label-outline label-red">POSTER</span></a><br/><br/>
</p> 

<p>
*Sun, S., *<u><b>Gill, M.L.</u></b>, Li, Y., Huang, M., Byrd, R.A. 
<a href="https://doi.org/10.1007/s10858-015-9923-x" target="_blank" style="text-decoration:none;"><span class="label label-outline label-purple">DOI</span></a>
<a href="publications/GillMichelle_JBNMR_2015.pdf" target="_blank" style="text-decoration:none;"><span class="label label-outline label-blue">PDF</span></a>
<a href="http://nestanmr.com/" target="_blank" style="text-decoration:none;"><span class="label label-outline label-orange">WEBSITE</span></a><br />
<b>Efficient and generalized processing of multidimensional NUS NMR Data: the NESTA algorithm and comparison of regularization terms</b><br/>
<i>Journal of Biomolecular NMR</i>, 2015, 62, 105–117<br/>
<i>* Authors contributed equally</i><br/>
Poster presented at <i>56<sup>th</sup> Experimental Nuclear Magnetic Resonance Conference</i>, 2015, Monterrey, CA
<a href="posters/GillMichelle_ENC_2015.pdf" target="_blank" style="text-decoration:none;"><span class="label label-outline label-red">POSTER</span></a><br/><br/>
</p>

<p>
<u><b>Gill, M.L.</u></b>, Byrd, R.A.
<a href="https://doi.org/10.1038/nsmb.2925" target="_blank" style="text-decoration:none;"><span class="label label-outline label-purple">DOI</span></a>
<a href="publications/GillMichelle_NSMB_2014.pdf" target="_blank" style="text-decoration:none;"><span class="label label-outline label-blue">PDF</span></a><br/>
<b>Dynamic activation of apoptosis: conformational ensembles of cIAP1 are linked to a spring-loaded mechanism</b><br/>
<i>Nature Structural Molecular Biology</i>, 2014, 21, 1022–1023<br/><br/>
</p>

<p>
<u><b>Gill, M.L.</u></b>, Palmer, A.G.
<a href="https://doi.org/10.1021/jp506580c" target="_blank" style="text-decoration:none;"><span class="label label-outline label-purple">DOI</span></a>
<a href="publications/GillMichelle_JPhysChemB_2014.pdf" target="_blank" style="text-decoration:none;"><span class="label label-outline label-blue">PDF</span></a><br/>
<b>Local isotropic diffusion approximation for coupled internal and overall molecular motions in NMR spin relaxation</b><br/>
<i>Journal of Physical Chemistry</i>, Series B, 2014, 118, 11120–11128<br/><br/>
</p>

<p>
Ergel, B., <u><b>Gill, M.L.</u></b>, Brown, L., Yu, B., Palmer, A.G., Hunt, J.F.
<a href="https://doi.org/10.1074/jbc.M114.575647" target="_blank" style="text-decoration:none;"><span class="label label-outline label-purple">DOI</span></a>
<a href="publications/GillMichelle_JBC_2014.pdf" target="_blank" style="text-decoration:none;"><span class="label label-outline label-blue">PDF</span></a><br/>
<b>Protein dynamics control the progression and efficiency of the catalytic reaction cycle of AlkB</b><br/>
<i>Journal of Biological Chemistry</i>, 2014, 289, 29584–29601<br />
Poster presented at <i>International Conference of Magnetic Resonance in Biological Sciences</i>, 2012, Lyon, France
<a href="posters/GillMichelle_ICMRBS_2012.pdf" target="_blank" style="text-decoration:none;"><span class="label label-outline label-red">POSTER</span></a><br/><br/>
</p>

<p>
<u><b>Gill, M.L.</u></b> and Palmer, A.G.
<a href="https://doi.org/10.1007/s10858-011-9533-1" target="_blank" style="text-decoration:none;"><span class="label label-outline label-purple">DOI</span></a>
<a href="publications/GillMichelle_JBNMR_2011.pdf" target="_blank" style="text-decoration:none;"><span class="label label-outline label-blue">PDF</span></a><br/>
<b>Multiplet-filtered and gradient-selected zero-quantum TROSY experiments for <sup>13</sup>C<sub>1</sub>H<sub>3</sub> methyl groups in proteins</b><br/>
<i>Journal of Biomolecular NMR</i>, 2011, 51, 245–251<br/>
Poster presented at <i>52nd Experimental Nuclear Magnetic Resonance Conference</i>, 2011, Monterrey, CA
<a href="posters/GillMichelle_ENC_2011.pdf" target="_blank" style="text-decoration:none;"><span class="label label-outline label-red">POSTER</span></a><br/><br/>
</p>

<p>
Ramsey, J.D., <u><b>Gill, M.L.</u></b>, Kamerzell, T.J., Price, E.S., Joshi, S.B., Bishop, S.M., Oliver, C.N., Middaugh, C.R.
<a href="https://doi.org/10.1002/jps.21619" target="_blank" style="text-decoration:none;"><span class="label label-outline label-purple">DOI</span></a>
<a href="publications/GillMichelle_JPharmSci_2009.pdf" target="_blank" style="text-decoration:none;"><span class="label label-outline label-blue">PDF</span></a><br/>
<b>Using empirical phase diagrams to understand the role of intramolecular dynamics in immunoglobulin G stability</b><br/>
<i>Journal of Pharmaceutical Sciences</i>, 2009, 98, 2432–2447<br/><br/>
</p>

<p>
<u><b>Gill, M.L.</u></b>, Strobel, S.A., and Loria, J.P.
<a href="https://doi.org/10.1093/nar/gkl616" target="_blank" style="text-decoration:none;"><span class="label label-outline label-purple">DOI</span></a>
<a href="publications/GillMichelle_NAR_2006.pdf" target="_blank" style="text-decoration:none;"><span class="label label-outline label-blue">PDF</span></a><br/>
<b>Crystallization and characterization of the thallium form of the <i>Oxytricha nova</i> G-quadruplex</b><br/>
<i>Nucleic Acids Research</i>, 2006, 34, 4506–4514<br/><br/>
</p>

<p>
<u><b>Gill, M.L.</u></b>, Strobel, S.A., and Loria, J.P.
<a href="https://doi.org/10.1021/ja055358f" target="_blank" style="text-decoration:none;"><span class="label label-outline label-purple">DOI</span></a>
<a href="publications/GillMichelle_JACS_2005.pdf" target="_blank" style="text-decoration:none;"><span class="label label-outline label-blue">PDF</span></a><br/>
<b><sup>205</sup>Tl NMR methods for the study of monovalent metal binding sites in nucleic acids</b><br/>
<i>Journal of the American Chemical Society</i>, 2005, 127, 16723–16732<br/>
Poster and selected oral presentation at <i>46th Experimental Nuclear Magnetic Resonance Conference</i>, 2005, Providence, RI<br/><br/>
</p>

<p>
Beach, H., Cole, R., <u><b>Gill, M.L.</u></b>, and Loria, J.P.
<a href="https://doi.org/10.1021/ja0514949" target="_blank" style="text-decoration:none;"><span class="label label-outline label-purple">DOI</span></a>
<a href="publications/GillMichelle_JACS_2005b.pdf" target="_blank" style="text-decoration:none;"><span class="label label-outline label-blue">PDF</span></a><br/>
<b>Conservation of µs-ms enzyme motions in the <i>apo-</i> and <i>substrate-mimicked</i> state</b><br/>
<i>Journal of the American Chemical Society</i>, 2005, 127, 9167–9176<br/><br/>
</p>

<p>
Adams, P.L., Stahley, M.R., <u><b>Gill, M.L.</u></b>, Kosek, A.B., Wang, J., and Strobel, S.A.
<a href="https://doi.org/10.1038/nature02642" target="_blank" style="text-decoration:none;"><span class="label label-outline label-purple">DOI</span></a>
<a href="publications/GillMichelle_RNA_2004.pdf" target="_blank" style="text-decoration:none;"><span class="label label-outline label-blue">PDF</span></a><br/>
<b>Crystal structure of a group I intron splicing intermediate</b><br/>
<i>RNA</i>, 2004, 12, 1867–1887<br/><br/>
</p>

<p>
Wiethoff, C.M., <u><b>Gill, M.L.</u></b>, Koe, G.S., Koe, J.G., and Middaugh, C.R.
<a href="https://doi.org/10.1002/jps.10391" target="_blank" style="text-decoration:none;"><span class="label label-outline label-purple">DOI</span></a>
<a href="publications/GillMichelle_JPharmSci_2003.pdf" target="_blank" style="text-decoration:none;"><span class="label label-outline label-blue">PDF</span></a><br/>
<b>A fluorescence study of the structure and accessibility of plasmid DNA condensed with cationic gene delivery vehicles</b><br/>
<i>Journal of Pharmaceutical Sciences</i>, 2003, 92, 1272–1285<br/><br/>
</p>

<p>
Wiethoff, C.M., <u><b>Gill, M.L.</u></b>, Koe, G.S., Koe, J.G., and Middaugh, C.R.
<a href="https://doi.org/10.1074/jbc.M207758200" target="_blank" style="text-decoration:none;"><span class="label label-outline label-purple">DOI</span></a>
<a href="publications/GillMichelle_JBC_2002.pdf" target="_blank" style="text-decoration:none;"><span class="label label-outline label-blue">PDF</span></a><br/>
<b>The structural organization of cationic lipid-DNA complexes</b><br/>
<i>Journal of Biological Chemistry</i>, 2002, 277, 44980–44987<br/><br/>
</p>

<p>
Silchenko, S., *<u><b>Sippel, M.L.</u></b>, Kuchment, O., Benson, D.R., Mauk, A.G., Altuve, A., and Rivera, M.
<a href="https://doi.org/10.1006/bbrc.2000.2968" target="_blank" style="text-decoration:none;"><span class="label label-outline label-purple">DOI</span></a>
<a href="publications/GillMichelle_BBRC_2000.pdf" target="_blank" style="text-decoration:none;"><span class="label label-outline label-blue">PDF</span></a><br/>
<b>Hemin is kinetically trapped in cytochrome b<sub>5</sub> from rat outer mitochondrial membrane</b><br/>
<i>Biochemical and Biophysical Research Communications</i>, 2000, 273, 467–472<br/>
<i>* M.L. Gill is formerly M.L. Sippel</i><br/>
</p>

</article>
</div>

<!-- Contact -->
<div class="wrapper style4 no-print">
<article id="contact" class="container 75%">
<header>
<h2>Contact Me</h2>

<ul class="social">
    <li><a href="mailto:michelle@michellelynngill.com" class="icon fa-envelope"><span class="label">Email</span></a></li>
    <li><a href="http://themodernscientist.com" class="icon fa-globe"><span class="label">Blog</span></a></li>
    <li><a href="https://www.github.com/mlgill" class="icon fa-github"><span class="label">Github</span></a></li>
    <li><a href="https://twitter.com/modernscientist" class="icon fa-twitter"><span class="label">Twitter</span></a></li>
    <li><a href="https://www.linkedin.com/in/michellelynngill" class="icon fa-linkedin"><span class="label">LinkedIn</span></a></li>

    <!-- <li><a href="#" class="icon fa-facebook"><span class="label">Facebook</span></a></li>
    <li><a href="#" class="icon fa-dribbble"><span class="label">Dribbble</span></a></li>
    <li><a href="#" class="icon fa-tumblr"><span class="label">Tumblr</span></a></li>
    <li><a href="#" class="icon fa-google-plus"><span class="label">Google+</span></a></li>
    <li><a href="#" class="icon fa-rss"><span>RSS</span></a></li>
    <li><a href="#" class="icon fa-instagram"><span>Instagram</span></a></li>
    <li><a href="#" class="icon fa-foursquare"><span>Foursquare</span></a></li>
    <li><a href="#" class="icon fa-skype"><span>Skype</span></a></li>
    <li><a href="#" class="icon fa-soundcloud"><span>Soundcloud</span></a></li>
    <li><a href="#" class="icon fa-youtube"><span>YouTube</span></a></li>
    <li><a href="#" class="icon fa-blogger"><span>Blogger</span></a></li>
    <li><a href="#" class="icon fa-flickr"><span>Flickr</span></a></li>
    <li><a href="#" class="icon fa-vimeo"><span>Vimeo</span></a></li> -->
    
</ul>
</header>

<!-- <div class="row">
    <div class="12u">
        <form method="post" action="#">
            <div>
                <div class="row">
                    <div class="6u 12u(mobile)">
                        <input type="text" name="name" id="name" placeholder="Name" />
                    </div>
                    <div class="6u 12u(mobile)">
                        <input type="text" name="email" id="email" placeholder="Email" />
                    </div>
                </div>
                <div class="row">
                    <div class="12u">
                        <input type="text" name="subject" id="subject" placeholder="Subject" />
                    </div>
                </div>
                <div class="row">
                    <div class="12u">
                        <textarea name="message" id="message" placeholder="Message"></textarea>
                    </div>
                </div>
                <div class="row 200%">
                    <div class="12u">
                        <ul class="actions">
                            <li><input type="submit" value="Send Message" /></li>
                            <li><input type="reset" value="Clear Form" class="alt" /></li>
                        </ul>
                    </div>
                </div>
            </div>
        </form>
    </div>
</div> -->

<footer>
    <ul><li>Last updated: 02/20/2022</li></ul>
    <ul id="copyright">
        <li>&copy; 2020 Michelle Lynn Gill</li><li>All rights reserved.</li><li>Design: <a href="https://html5up.net">HTML5 UP</a></li>
    </ul>
</footer>
</article>
</div>

<!-- Scripts -->
<script src="assets/js/jquery.min.js"></script>
<script src="assets/js/jquery.scrolly.min.js"></script>
<script src="assets/js/skel.min.js"></script>
<script src="assets/js/skel-viewport.min.js"></script>
<script src="assets/js/util.js"></script>
<!--<script src="assets/js/kube.min.js"></script>-->
<!--[if lte IE 8]><script src="assets/js/ie/respond.min.js"></script><![endif]-->
<script src="assets/js/main.js"></script>

</body>
</html>
